code/modules/integrated_electronics/passive/power.dm

```diff
@@ -156,7 +156,7 @@
 /obj/item/integrated_circuit/passive/power/relay/large
 	name = "large tesla power relay"
 	desc = "A seemingly enigmatic device which connects to nearby APCs wirelessly and draws power from them, now in industiral size!"
-	w_class = WEIGHT_CLASS_BULKY
+	w_class = WEIGHT_CLASS_NORMAL
 	extended_desc = "The siphon generates 2 kW of energy, so long as an APC is in the same room, with a cell that has energy.  It will always drain \
```

Comment:
```suggestion
	extended_desc = "The siphon draws 2 kW of electrical energy, as long as an APC is in the same room, with a cell that has energy.  It will always drain \
```